.travis.yml

```diff
@@ -1,3 +1,8 @@
+os: linux
+dist: focal
+group: edge
+arch: arm64-graviton2
```

Comment:
Thought I'd try out the new AWS Graviton2 CPUs. The start-up time is faster than before, but the tests still take as long as they needed before.

Reply:
The build website ought to be on travis-ci.com. There's also a [migration page](https://docs.travis-ci.com/user/migrate/open-source-repository-migration), but all you need to do is to change the link of the build status in the readme file and set the env vars on travis-ci.com